source/utils/regex.ts

```diff
@@ -0,0 +1,2 @@
+export const base64Regex =
```

Comment:
already exported in sysweb3 utils

Reply:
I find it odd to import a simple regex from a lib that is supposed to handle sys and web3. It's harder to maintain. I just moved this out of the file were it was

Reply:
we have an utils package for this reason